components/_patterns/00-base/global/base/_pl-base.scss

```diff
@@ -16,11 +12,11 @@
     font-size: 3rem;
     text-transform: uppercase;
     background-color: $gray-darker;
-    margin: 0 calc(-50vw + 50%) $space calc(-50vw + 50%);
 
     a {
       @include wrapper(
-        $v-padding: $space
+        $v-padding: $space,
+        $h-padding: $space / 2
```

Comment:
This caused weird issues in my testing. Let's remove line 19, and this is good to go